she317ryl-mangalore.md

```diff
@@ -0,0 +1,10 @@
+# Mangalore, Karnataka
+### Restaurants
+- Mangalore Pearl
```

Comment:
- What type of food is served at Mangalore Pearl? 
- Is it more extensive than indian cuisine?